### DISCUSSION TITLE
Tab plugin navigation

### DISCUSSION THREAD

**Discussion by E-JOKE** [Budibase/budibase#9799] [Help] [answered]
When a screen with a tab container (plugin) is opened, it goes straight to the first tab. Doen anyone know if it is posible make an action which opens another (not the first) tab?

**Comment by andz-bb** [comment, marked as answer]
Hey @E-JOKE - I just made a fork of the Super Tabs community component with an update that allows you to configure which tab appears first on a screen load. I've made a PR into the main plugin but if you want to try it in the meantime you can give it a go here: https://github.com/andz-bb/bb-component-SuperTabs

**Reply by poirazis** [reply]
Hey @andz-bb and @E-JOKE 
the PR has been merged into the main plugin. You can update your plugin to v.1.1.1 to get default Tab index selection.

A huge thank you for the PR @andz-bb !
